web/src/tippyjs.js

```diff
@@ -317,44 +345,25 @@ export function initialize() {
         appendTo: () => document.body,
     });
 
-    delegate("body", {
-        target: ".recipient_bar_icon",
-        onShow(instance) {
-            if (!document.body.contains(instance.reference)) {
-                return false;
-            }
-            const $elem = $(instance.reference);
-
-            const config = {attributes: false, childList: true, subtree: true};
-            const target = $elem.parents(".message_header.message_header_stream.right_part").get(0);
-            const nodes_to_check_for_removal = [
-                $elem.parents(".recipient_bar_controls").get(0),
-                $elem.get(0),
-            ];
-            hide_tooltip_if_reference_removed(target, config, instance, nodes_to_check_for_removal);
-            return true;
-        },
+    // In case of recipient bar icons, following change
+    // ensures that tooltip doesn't hide behind the message
+    // box or it is not limited by the parent container.
+    message_list_tooltip([".recipient_bar_icon"], {
+        delay: LONG_HOVER_DELAY,
```

Comment:
We don't need the extra delay here, also the comment seems unnecessary.

Reply:
The delay was there in the case of delegate so, I left it untouched, however I have removed it after your suggestion.